test/e2e/e2e_test.go

```diff
@@ -43,12 +43,12 @@ func init() {
 
 	prfs, _ := c.ListProfiles()
 	if len(prfs) == 0 {
-	        fmt.Println("Start creating profile...")
+		fmt.Println("Start creating profile...")
 		var body = &model.ProfileSpec{
 			Name:        "default",
 			Description: "default policy",
 		}
-                prf, err := c.CreateProfile(body)
+		prf, err := c.CreateProfile(body)
```

Comment:
Same here

Reply:
I don't know why, it's probably because of the IDE.